manifests/init.pp

```diff
@@ -72,8 +72,12 @@
 #   Set to true if corosync_votequorum should be used as quorum provider.
 #   Defaults to false.
 #
+# [*set_expected_votes*]
+#   How many votes required to create a quorum?? (How many nodes assuming each node is 1 vote)
```

Comment:
Could this be "Number of votes required to create a quorum, or the number of nodes needed if each node has 1 vote"